app/src/marketplace/containers/ProductPage/useDataUnionStats.js

```diff
@@ -1,5 +1,3 @@
-// @flow
```

Comment:
Should we keep `@flow` but suppress specific errors we're getting in this file?

Reply:
I think we decided to phase out flow slowly and then introduce something else like Typescript. Flow has been already removed for some other components, also here I decided to just get rid of it instead of trying to please the cryptic Flow demands

Reply:
Ah, that makes total sense. Typescript would be nice to have 👍